src/main/extension-filesystem.ts

```diff
@@ -1,17 +1,19 @@
 import { randomBytes } from "crypto"
 import { SHA256 } from "crypto-js"
 import { app } from "electron"
-import { ensureDir, mkdir, pathExists } from "fs-extra"
+import fse from "fs-extra"
 import { action, observable, toJS } from "mobx"
 import path from "path"
 import { BaseStore } from "../common/base-store"
 
+type LensExtensionId = string;
```

Comment:
This should be exported from `lens-extension.ts` as `type` (already defined there)

Reply:
Ah fair, thanks